qhub/template/stages/01-terraform-state/aws/modules/terraform-state/main.tf

```diff
@@ -1,27 +1,27 @@
-resource "aws_s3_bucket" "terraform-state" {
-  bucket = "${var.name}-terraform-state"
-  acl    = "private"
+# resource "aws_s3_bucket" "terraform-state" {
+#   bucket = "${var.name}-terraform-state"
+#   acl    = "private"
 
-  force_destroy = true
+#   force_destroy = true
 
-  versioning {
-    enabled = true
-  }
+#   versioning {
+#     enabled = true
+#   }
 
-  tags = merge({ Name = "S3 remote terraform state store" }, var.tags)
-}
+#   tags = merge({ Name = "S3 remote terraform state store" }, var.tags)
+# }
 
-resource "aws_dynamodb_table" "terraform-state-lock" {
-  name = "${var.name}-terraform-state-lock"
+# resource "aws_dynamodb_table" "terraform-state-lock" {
+#   name = "${var.name}-terraform-state-lock"
 
-  read_capacity  = 1
-  write_capacity = 1
-  hash_key       = "LockID"
+#   read_capacity  = 1
+#   write_capacity = 1
+#   hash_key       = "LockID"
 
-  attribute {
-    name = "LockID"
-    type = "S"
-  }
+#   attribute {
+#     name = "LockID"
+#     type = "S"
+#   }
 
-  tags = merge({ Name = "DynamoDB table for locking terraform state store" }, var.tags)
-}
+#   tags = merge({ Name = "DynamoDB table for locking terraform state store" }, var.tags)
+# }
```

Comment:
Why this code is commented on? AFAIK this is needed to store the terraform state files for each stage resource.

Reply:
This was just a dummy change. I will remove this on the final stage of PR review. This PR comment was generated via this change: https://github.com/Quansight/qhub/pull/1316#issuecomment-1154863716

Reply:
Removed these changes 👍